### ISSUE TITLE
Not working under perl 5.26.0 or newer.

### ISSUE THREAD

**Issue by gugod** [dolmen/angel-PS1#23]
When running the fatpacked version of angel-PS1, it dies with the following error -- if the invoking perl is newer than 5.26.0 

```
% eval `./angel-PS1`
angel-PS1: Unimplemented: POSIX::tmpnam(): use File::Temp instead at /Users/st21277/bin/angel-PS1 line 2856.
```

Some additional info:

```
% which perl
/Users/gugod/perl5/perlbrew/perls/perl-5.26.1/bin/perl
% echo $SHELL
/bin/zsh
```

The subroutine `POSIX::tmpnam()` was simply removed. See: [https://perldoc.perl.org/perl5260delta.html#POSIX%3a%3atmpnam()-has-been-removed](https://perldoc.perl.org/perl5260delta.html#POSIX%3a%3atmpnam()-has-been-removed)

**Comment by dolmen** [comment]
Fixed by bde8772787528d691906957186bb5c5d071a2632 in `devel` branch.

**Comment by dolmen** [comment]
Fix released with v0.97.
